src/AppBundle/Service/Restore/RestoreService.php

```diff
@@ -42,14 +42,14 @@ public function getFilesInBackupForTimestamp(Backup $backup)
 
         $remoteBackupPath = $backupDestination->getDestinationText().$backupSchedule->getName();
 
-        $result = shell_exec('duplicity list-current-files --time '.date_format($backup->getTimestamp(), DATE_ISO8601).' '.$remoteBackupPath);
+        exec('duplicity list-current-files --time '.date_format($backup->getTimestamp(), DATE_ATOM).' '.$remoteBackupPath.' 2>&1 &', $result);
```

Comment:
@janritter ist es absicht, dass das ergebnis nicht mehr in $result gepeichert wird?

Reply:
Bei exec wird das Ergebnis in die übergebene variable result gespeichert